cron/hourly/git_maintenance_tasks_lock

```diff
@@ -4,4 +4,5 @@ set -e
 # Run git_maintenance_tasks only if not running already
 DIR="$(readlink -e "$(dirname $0)/..")"
 mkdir -p ~/lmms-repo
-flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock "$DIR/git_maintenance_tasks"
+flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock \
+	"$DIR/git_maintenance_tasks_retry"
```

Comment:
Thanks for the explanation.

> One script should do one job and do it well. Technically, you can merge the scripts, but with a maintenance penalty; do you prefer that?

I would prefer we step back as originally proposed and [don't use lock files at all](https://github.com/LMMS/lmms.io/pull/198#issuecomment-277459438) .  This would allow self-awareness and atomic operation but without the limitations of passing a command to `flock`, right?